### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GaddamADF**
Pls test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # FabricDemo
```
Comment (GVenkatMSFabric) [inline README.md:1]: Why you changed the code pls explain

Comment (GVenkatMSFabric) [reply README.md:1]: Because some development point of view its required.

Comment (GaddamADF) [reply README.md:1]: Because currently we are working on Development Point of view.

**Review by GVenkatMSFabric — APPROVED** [review]
Approved the below change request.

**Pull request merged**